.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push]
```

Comment:
does this run on all branches including PRs?  Because on a PR branch I could see changing a chart before going back to master?

Reply:
Oh, this should only run on branches (and not `gh-pages`), not on `master`, because it will fail on master as it races with release. Will fix.